### DISCUSSION TITLE
what do you mean by work eligibility

### DISCUSSION THREAD

**Discussion by Sajiyah-Salat** [community/community#49889] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I try to **explore octernships** it says, you currently does not meet the criteria. However I am a active contributor to k8s repo. I am 20 year student in manipal university. I dont know why I am unable to see the project lists. what do you mean by work eligibility. I mean In india 18+ year student are eligible for internships. let me know what I am missing. Thank you.

**Comment by Sajiyah-Salat** [comment]
> Hi @Sajiyah-Salat!
> 
> Good luck with your quest to become an Octernships intern!
> 
> Octernships is a program by OctoML that offers internships to students who contribute to open-source projects. Although you're an active contributor to the Kubernetes repo, you might not meet all the eligibility criteria for Octernships. These criteria could be related to your location, citizenship, or work authorization status.
> 
> To find out more about the eligibility criteria, you can check out Octernships' website or reach out to OctoML's support team. "Work eligibility" refers to whether someone is legally allowed to work in a specific country or region, based on factors like citizenship, age, or educational qualifications. But keep in mind that specific internship requirements can vary by company and location.

But I am an indian and octernships allow indian students.

**Comment by ashwinexe** [comment]
Hey there, you have to be _an active contributor on GitHub_ to unlock this opportunity. More on what that means in [this FAQ](https://github.com/orgs/community/discussions/49380)
